app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -193,8 +196,8 @@ const getActualAcuantCamera = (): AcuantCameraInterface => {
 };
 
 function AcuantContextProvider({
-  sdkSrc = '/acuant/11.7.1/AcuantJavascriptWebSdk.min.js',
-  cameraSrc = '/acuant/11.7.1/AcuantCamera.min.js',
+  sdkSrc = '/acuant/11.7.0/AcuantJavascriptWebSdk.min.js',
+  cameraSrc = '/acuant/11.7.0/AcuantCamera.min.js',
```

Comment:
Are we planning to keep this code pattern around for a while?

If so, and if we plan to hard-code the version sources, I'd suggest getting rid of the prop altogether, since it's misleading to imply that it can be customized if the logic of the component will ignore the value depending on the A/B testing value.

Alternatively, we could adapt the component to receive the two sets of sources.

Reply:
> Are we planning to keep this code pattern around for a while?
  
My understanding is that we wanted to let this run for a week, maybe two. However I think we should have clarification. If we are going to AB test upgrades in the future (and why not?) then you are right, we should change some if these assumptions. Additionally, we'd probably want to set the old/new version numbers in the identitystore config